pkg/kubelet/apis/config/types.go

```diff
@@ -721,6 +742,11 @@ type ServiceAccountTokenAttributes struct {
 	// +required
 	ServiceAccountTokenAudience string
 
+	// cacheType indicates the type of cache key use for caching the credentials returned by the plugin
+	// when the service account token is used.
```

Comment:
Can give guidance here to use Token if you don't know what you're doing

"The most conservative option is to set to "Token", which ..."